spec/api/v1/rolls_request_spec.rb

```diff
@@ -0,0 +1,85 @@
+require 'rails_helper'
+
+describe "Roller API" do
+
+  before(:each) do
+    @roll = Roll.create(title: 'Rick Roll', start_time: 19, end_time: 23, youtube_indentifer: 'lYbYoaBrXpU')
+    @fenton_roll = Roll.create(title: 'Rick Roll', start_time: 19, end_time: 23, youtube_indentifer: '3GRSbr0EYYU')
+  end
+
+
+
+  describe "get /rolls" do
+    context "is valid" do
+      it 'returns a JSON collection of all of the rolls' do
+        get '/api/v1/rolls'
+        response_body = JSON.parse(response.body)
+        expect(response).to be_success
+        expect(response_body.length).to eq(2)
+      end
+    end
+
+    context "is invalid" do
```

Comment:
We can't "get" invalid rolls -- they shouldn't be in the database to begin with because validation happens when things are put into the database (creating or updating)